src/group_theory/p_group.lean

```diff
@@ -212,13 +212,13 @@ begin
   exact ⟨j + k, by rwa [subtype.ext_iff, (H.comap ϕ).coe_pow]⟩,
 end
 
+lemma ker_is_p_group_of_injective {K : Type*} [group K] {ϕ : K →* G} (hϕ : function.injective ϕ) :
```

Comment:
I don't understand this lemma: is it saying that the trivial subgroup is a `p`-group?

Reply:
Yes, that's what the lemma boils down to. But this specific formulation has come up a few times now, so I thought it might be a good idea to give it its own lemma.

Reply:
I see... I will let someone more experienced have a look, since the statement is a little weird.